builds/android/AndroidManifest.xml

```diff
@@ -4,8 +4,8 @@
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
       package="org.easyrpg.player"
-      android:versionCode="1"
-      android:versionName="1.0"
+      android:versionCode="2533"
+      android:versionName="0.2.2.70"
```

Comment:
Yep I forgot to ignore this file in the first commit, should I have to do anything ?